docs/README.md

```diff
@@ -412,8 +412,406 @@ Steps:
     ```
     The old `IstioRevision` resource and the old control plane will be deleted when the grace period specified in the `Istio` resource field `spec.updateStrategy.inactiveRevisionDeletionGracePeriodSeconds` expires.
 
-## Multi-cluster
-tbd
+## Multicluster
```

Comment:
Should this be "Multi-cluster"?

Reply:
Either way is fine with me as long as we're consistent